<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDataDialog</class>
 <widget class="QDialog" name="ConnectionDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новое подключение</string>
  </property>
  <widget class="QLabel" name="userLinelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>231</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Имя пользователя для подключения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>370</y>
     <width>130</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Reference Sans Serif</family>
    </font>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="portLineLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Порт сервера базы данных</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="databaseLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="databaseLinelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Имя базы данных</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLinelabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>251</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пароль пользователя для подключения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hostLineLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Адрес сервера базы данных</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="portLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
